### ISSUE TITLE
Please implement auto-discovery

### ISSUE THREAD

**Issue by GermanBluefox** [iobroker-community-adapters/ioBroker.sma-em#15]
https://github.com/ioBroker/ioBroker.discovery/tree/master/lib/adapters

**Comment by DutchmanNL** [comment]
wil check if this can be done, in basic its announcing by UDP

**Comment by pdbjjens** [comment]
@GermanBluefox @DutchmanNL 
I have implemented a detector module for this adapter sma-em. I also developed a method for SMA Speedwire discovery. They are ready for a PR against the ioBroker.discovery adapter. 
However, since these are my first discovery modules, you might want to have a look at [https://github.com/pdbjjens/ioBroker.discovery](https://github.com/pdbjjens/ioBroker.discovery) and send some comments for improvement before I issue a PR.

FYI: the speedwire method is not needed by the sma-em detector module (it has the type "once" since it can detect all energy meter IPs on it's own by listening to the multicast messages of the energy meters just once for a period of 2 s) 
The sma-em detector module requires the not-yet-released version 1.0.0 of the sma-em (see: [https://github.com/iobroker-community-adapters/ioBroker.sma-em](https://github.com/iobroker-community-adapters/ioBroker.sma-em)
The speedwire method might be useful for other SMA adapters like ioBroker.speedwire.
